lib/inspect_rpmdeps.c

```diff
@@ -260,6 +263,14 @@ static bool check_explicit_lib_deps(struct rpminspect *ri, Header h, deprule_lis
                 continue;
             }
 
+            /*
+             * issue 1246: Prevent false positives when the library is provided
+             * by a subpackage coming from another SRPM.
+             */
+            peer_srpm = headerGetString(peer->after_hdr, RPMTAG_SOURCERPM);
+            if (peer_srpm && strcmp(peer_srpm, srpm)) {
```

Comment:
I think this is a suitable workaround now until I can get better cross-RPM dependency handling for modules.  I would prefer that we also guard this test to ri->buildtype as KOJI_BUILD_MODULE.

Reply:
Thanks for checking. That guard makes sense, I've updated the PR with the guard.